lib/services/local/sessiontracker.go

```diff
@@ -123,7 +123,15 @@ func (s *sessionTracker) GetActiveSessionTrackers(ctx context.Context) ([]types.
 			return nil, trace.Wrap(err)
 		}
 
-		sessions[i] = session
+		if session.Expiry().After(time.Now().UTC()) {
+			sessions[i] = session
+		} else if item.Expires.IsZero() { // Check if the expiry is not set.
+			if err := s.bk.Delete(ctx, item.Key); err != nil {
+				if !trace.IsNotFound(err) {
+					return nil, trace.Wrap(err)
```

Comment:
Failing the caller because we couldn't clear an expired item seems a bit strange. Nothing in the signature or godoc of the method would suggest that to me.

Reply:
Moved this into a seperate goroutine that just logs errors, helps improve latency as well if there are trackers to clear.